api/src/main/java/org/motechproject/nms/api/web/KilkariController.java

```diff
@@ -62,7 +63,7 @@ public InboxResponse getInboxDetails(@RequestParam(required = false) Long callin
             throw new NotFoundException(String.format(NOT_FOUND, "callingNumber"));
         }
 
-        Set<Subscription> subscriptions = subscriber.getSubscriptions();
+        Set<Subscription> subscriptions = subscriber.getActiveSubscriptions();
```

Comment:
If we end up doing Early Subscriptions then this will have to change.  I think it is ok for now.